cmd/index.js

```diff
@@ -73,6 +73,7 @@ module.exports = async (ipc, argv = Bare.argv.slice(1)) => {
     flag('--name <name>', 'Advanced. Override app name'),
     flag('--json', 'Newline delimited JSON output'),
     flag('--encryption-key <name>', 'Application encryption key'),
+    flag('--no-ask', 'Suppress permissions dialog'),
```

Comment:
this change is also in https://github.com/holepunchto/pear/pull/283/ - dialogs not dialog